qtox.pro

```diff
@@ -272,135 +230,133 @@ unix:!macx:!android {
     }
 }
 
-!android {
-    RESOURCES += res.qrc \
-        smileys/smileys.qrc
-
-    HEADERS  += \
-        src/friend.h \
-        src/group.h \
-        src/grouplist.h \
-        src/friendlist.h \
-        src/persistence/smileypack.h \
-        src/widget/emoticonswidget.h \
-        src/widget/style.h \
-        src/widget/tool/croppinglabel.h \
-        src/widget/maskablepixmapwidget.h \
-        src/video/videosurface.h \
-        src/net/toxuri.h \
-        src/net/toxdns.h \
-        src/persistence/toxsave.h \
-        src/persistence/serialize.h \
-        src/chatlog/chatlog.h \
-        src/chatlog/chatline.h \
-        src/chatlog/chatlinecontent.h \
-        src/chatlog/chatlinecontentproxy.h \
-        src/chatlog/content/text.h \
-        src/chatlog/content/spinner.h \
-        src/chatlog/content/filetransferwidget.h \
-        src/chatlog/chatmessage.h \
-        src/chatlog/content/image.h \
-        src/chatlog/customtextdocument.h \
-        src/widget/form/settings/aboutform.h \
-        src/widget/form/settings/advancedform.h \
-        src/chatlog/content/notificationicon.h \
-        src/chatlog/content/timestamp.h \
-        src/chatlog/documentcache.h \
-        src/chatlog/pixmapcache.h \
-        src/persistence/offlinemsgengine.h \
-        src/widget/form/addfriendform.h \
-        src/widget/form/chatform.h \
-        src/widget/form/groupchatform.h \
-        src/widget/form/settingswidget.h \
-        src/widget/form/settings/genericsettings.h \
-        src/widget/form/settings/generalform.h \
-        src/widget/form/settings/privacyform.h \
-        src/widget/form/settings/avform.h \
-        src/widget/form/filesform.h \
-        src/widget/form/profileform.h \
-        src/widget/tool/chattextedit.h \
-        src/widget/tool/friendrequestdialog.h \
-        src/widget/friendwidget.h \
-        src/widget/groupwidget.h \
-        src/widget/widget.h \
-        src/video/netcamview.h \
-        src/widget/friendlistwidget.h \
-        src/widget/genericchatroomwidget.h \
-        src/widget/form/genericchatform.h \
-        src/widget/tool/adjustingscrollarea.h \
-        src/widget/form/loadhistorydialog.h \
-        src/widget/form/setpassworddialog.h \
-        src/widget/form/tabcompleter.h \
-        src/net/autoupdate.h \
-        src/widget/tool/callconfirmwidget.h \
-        src/widget/systemtrayicon.h \
-        src/widget/qrwidget.h \
-        src/widget/systemtrayicon_private.h \
-        src/widget/loginscreen.h \
-        src/ipc.h
+RESOURCES += res.qrc \
+    smileys/smileys.qrc
+
+HEADERS  += \
```

Comment:
Either add a comment that here's the stuff that is not portable/useful on android, or merge it with `HEADERS` below.